src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -303,12 +316,19 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
                     } elseif ($discoverSymbolsPhase && !\defined('T_ATTRIBUTE') && $tokens[$index]->isComment() && Preg::match('/#\[\s*('.self::REGEX_CLASS.')/', $tokens[$index]->getContent(), $matches)) { // @TODO: drop when PHP 8.0+ is required
                         $this->determineShortType($matches[1], $uses, $namespaceName);
                     } elseif ($tokens[$index]->isGivenKind(T_DOC_COMMENT)) {
+                        Preg::matchAll('/\*\h*@(?:psalm-|phpstan-)?(?:template(?:-covariant|-contravariant)?|(?:import-)?type)\h+('.TypeExpression::REGEX_IDENTIFIER.')(?!\S)/i', $tokens[$index]->getContent(), $matches);
```

Comment:
`psalm-`/`phpstan-` prefixes and `-covariant`/`-contravariant` suffixes are not covered with tests. Maybe it's possible to make simple, single test case for them?

Reply:
I will add some tests with `leading_backslash_in_global_namespace` for these.